content/docker-hub/release-notes.md

```diff
@@ -12,6 +12,10 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 2023-08-28
+
+- Organizations with SSO enabled can assign members to roles, organizations, and teams with[SCIM role mapping](scim.md#set-up-role-mapping). 
```

Comment:
This link actually generated correctly without the space. Seems it's not really necessary.

```suggestion
- Organizations with SSO enabled can assign members to roles, organizations, and teams with [SCIM role mapping](scim.md#set-up-role-mapping). 
```

Reply:
Ha, that's weird and unintentional! I guess the [link renderer](https://github.com/docker/docs/blob/main/layouts/_default/_markup/render-link.html) inserts a whitespace or newline, which for HTML paragraphs means a space gets inserted.